beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -439,6 +446,32 @@ func VoluntaryExitFromConsensus(e *eth.VoluntaryExit) *VoluntaryExit {
 	}
 }
 
+func (m *SyncCommitteeMessage) ToConsensus() (*eth.SyncCommitteeMessage, error) {
+	slot, err := strconv.ParseUint(m.Slot, 10, 64)
+	if err != nil {
+		return nil, NewDecodeError(err, "Slot")
+	}
+	root, err := hexutil.Decode(m.BeaconBlockRoot)
```

Comment:
we'll need to remember to update this when the deneb-produce blockv3 pr merges in